modules/internal/ChapelSyncvar.chpl

```diff
@@ -111,18 +111,30 @@ module ChapelSyncvar {
     var  wrapped : getSyncClassType(valType) = nil;
     var  isOwned : bool                      = true;
 
-    proc _syncvar(type valType) {
+    proc init(type valType) {
       ensureFEType(valType);
-
-      wrapped = new (getSyncClassType(valType))(valType);
-      isOwned = true;
-    }
-
-    proc _syncvar(type valType, const other : _syncvar(valType)) {
-      ensureFEType(valType);
-
-      wrapped = other.wrapped;
-      isOwned = false;
+      this.valType = valType;
+      this.wrapped = new (getSyncClassType(valType))(valType);
+      this.isOwned = true;
```

Comment:
You could omit the initialization of isOwned, I think

Reply:
Omit because of default initialization?

Reply:
Yeah.  This is the same as the declared default value for the field, so omitted initialization would just insert the same line